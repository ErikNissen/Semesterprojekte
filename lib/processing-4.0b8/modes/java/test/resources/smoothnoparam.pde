version https://git-lfs.github.com/spec/v1
oid sha256:3963ee06bb0bfaedb4ef552c51cd518d8a9e5465a883029374003278b22027d7
size 173
