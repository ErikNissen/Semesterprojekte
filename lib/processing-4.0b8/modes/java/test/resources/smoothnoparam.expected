version https://git-lfs.github.com/spec/v1
oid sha256:1957958a6d46799b15c8c6abaf0c053fc51b4d1d6cf271928e8cc165b037ff1f
size 964
